src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -85,8 +85,9 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
```

Comment:
Well, I don't know. We start this debate over here: https://github.com/symfony/symfony/pull/14132#discussion_r27778413, but no result for now.

I'm not convinced, as SymfonyStyle acts as a pseudo-decorator, and fully replace the output for the whole command (IMO it's even better, by avoiding using the non-decorated output by error and "mix styles").